template-for-repository/proofs/Makefile.common

```diff
@@ -300,54 +273,137 @@ CBMC_REMOVE_FUNCTION_BODY := $(patsubst %,--remove-function-body %, $(REMOVE_FUN
 
 # Compile project sources
 $(PROJECT_GOTO)1.goto: $(PROJECT_SOURCES)
-	mkdir -p $(dir $@)
-	mkdir -p $(dir $(call LOG_FROM_ENTRY,$@))
-	$(call DO_GOTO_CC,--export-file-local-symbols $(COMPILE_FLAGS) $(INCLUDES) $(DEFINES),$^,$@)
+	$(LITANI) add-job \
+	  --command \
+	    '$(GOTO_CC) $(CBMC_VERBOSITY) --export-file-local-symbols $(COMPILE_FLAGS) $(INCLUDES) $(DEFINES) $^ -o $@' \
+	  --inputs $^ \
```

Comment:
It was a mistake to compile all inputs at once in the original template.  Doing that here may be too much to do in a single pull request.  But is it going to be a problem to fix this in the next pull request?

Reply:
As above, I would like to restrict this PR to purely porting to Litani.